recipes/libspnav/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "libspnav" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "7ae4d7bb7f6a5dda28b487891e01accc856311440f582299760dace6ee5f1f93" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}.tar.gz
```

Comment:
Having `source/fn` is uncommon by now and should only be used if necessary. For an explanation see https://github.com/conda-forge/conda-forge.github.io/pull/709/files#diff-29e8b4bf4d69b118f7b3c877f2f39478R29.